.github/workflows/ci.yml

```diff
@@ -85,8 +84,12 @@ jobs:
           python -m pip install --upgrade pip wheel
           pip install -r requirements_test.txt
         name: 'Install dependencies with pip'
-      - run: pip install ${{ matrix.qt }}
+      - run: pip install PyQt5
         name: 'Install Qt binding'
+        if: matrix.os == 'macos-latest'
+      - run: pip install PySide2
+        name: 'Install Qt binding'
+        if: matrix.os != 'macos-latest'
```

Comment:
Any way to keep testing pyside2 on Linux as well? Maybe the matrix should be macos-pyqt5, linux-pyqt5, linux-pyside2?

Reply:
PySide2 is still tested on linux and windows for PyPI jobs.
PyQt5 is set for conda jobs though.

Reply:
Oh okay, I misread the config -- great!